### PR TITLE
add docs

### LINKED ISSUE #57: Hello-world doc and AWS KMS

### PR DESCRIPTION
**Pull request by mrandi**
fixes #57

### CODE REVIEW

File: user-guide/hello-world.rst [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    $ aws s3 ls s3://mint-example-bucket
+    # there should be a new folder for your application
+
+Deploy!
+
+This will configure your system:
+
+.. code-block:: bash
+
+    $ stups configure
```
Comment (hjacobs) [inline user-guide/hello-world.rst]: this needs to be done before using "pierone"

File: user-guide/hello-world.rst [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+* Taupage pushes Taupage config userdata to fullstop.
+* Taupage pulls Docker image from Pier One using the app credentials
+* Taupage starts the Docker container
+* Taupage signals CFN
+
+Wait for completion by watching the Senza status output.
+
+.. code-block:: bash
+
+    $ senza status deploy-definition.yaml -w 2
```
Comment (hjacobs) [inline user-guide/hello-world.rst]: you can use uppercase "-W" to wait with the default sleep time

File: user-guide/hello-world.rst [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+.. Important::
+
+    In case of error go to your log provider, if you did not configure it.
+    Go in aws, EC2 service, find your instance, right click, Instance Settings, Get System Log
+
+Test stack.
+
+.. code-block:: bash
+
+    $ curl -v http://<address>:8000/index.html?schema=schema/stups.json
```
Comment (hjacobs) [inline user-guide/hello-world.rst:180]: How can we access the address if the ELB is internal only?

File: user-guide/hello-world.rst [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+    This will not works! Because of the missing OAUTH2 we have created an internal LB.
+    To test it we will need to :ref:`follow the same guide as for a DB connection <dig-a-tunnel>` and than try again.
+
+Get instance IP:
+
+.. code-block:: bash
+
+    $ senza instances zalando-cheat-sheet-generator
+
+Let us piu to the Odd bation host:
```
Comment (hjacobs) [inline user-guide/hello-world.rst]: bation -> bastion

**Comment by prayerslayer** [comment]
:+1:

**Pull request merged**